system/lib/libc/emscripten_syscall_stubs.c

```diff
@@ -28,7 +28,7 @@ static int g_pid = 42;
 static int g_pgid = 42;
 static int g_ppid = 1;
 static int g_sid = 42;
-static mode_t g_umask = S_IRWXU | S_IRWXG | S_IRWXO;
+static mode_t g_umask = S_IWGRP | S_IWOTH;
```

Comment:
For reference, here are the definitions of values used here:
https://github.com/emscripten-core/emscripten/blob/8eb432e21ba19b3f14b2b367891c03f704ad45bf/system/lib/libc/musl/include/sys/stat.h#L60-L74